starters/deno-oak-denodb/src/graphql/resolvers/query_handler.ts

```diff
@@ -1,11 +1,34 @@
+import { Cache } from '../../cache/cache.ts';
 import { Technologies } from '../../db/model/technology.ts';
 
-export const getTechnologies = async () => {
-	const technologies = await Technologies.all();
-	return technologies;
+export const getTechnologies = async (
+	_p: unknown,
+	_: unknown,
+	{ ds }: { ds: Cache },
+	info: Record<string, string>,
+): Promise<Technologies[]> => {
+	const cacheTechnologies = await ds.readItem<Technologies[]>({ cacheKey: info.fieldName });
+	if (cacheTechnologies) {
+		return cacheTechnologies;
+	}
```

Comment:
This "read item from cache an check if it is cached" routine is very repetitive. I know it is now only two endpoints, but a generic higher-order function might be good to have. WDYT?